src/masternodes/mn_checks.cpp

```diff
@@ -843,6 +754,40 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         return !res ? res : mnview.ResignMasternode(obj, tx.GetHash(), height);
     }
 
+    Res operator()(const SetForcedRewardAddressMessage& obj) const {
+        auto const node = mnview.GetMasternode(obj.nodeId);
+        if (!node) {
+            return Res::Err("%s: node %s does not exist", __func__, obj.nodeId.ToString());
```

Comment:
Remove all references to `__func__` it will be `operator()` caller will add function name automatically.